### PR TITLE
Grammar and spelling fixes

### PR DESCRIPTION
**Pull request by donno2048**

### CODE REVIEW

**Comment by Akemi** [comment]
i would prefer to have all the grammar and man changes in one PR and maybe even on one commit. there are potential some controversial changes so it is most likely better to have a discussion in one place.

**Comment by donno2048** [comment]
> i would prefer to have all the grammar and man changes in one PR

Done.

File: DOCS/contribute.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Sending patches
   ``git format-patch``.
 - Plain diffs posted as pastebins are not acceptable! (Especially if the http
   link returns HTML.) They only cause extra work for everyone, because they lack
-  commit message and authorship information.
-- Never send patches to any of the developers email addresses.
+  commit messages and authorship information.
```
Comment (Akemi) [inline DOCS/contribute.md:18]: wouldn't it "authorship informations" too?

File: DOCS/waf-buildsystem.rst [hunk 2]
```diff
@@ -68,7 +68,7 @@ consists of running ``pkg-config`` and looking for ``vdpau`` with version
 ``>= 0.2``. If the check succeeds a ``#define HAVE_VDPAU 1`` will be added to
 ``config.h``, if not ``#define HAVE_VDPAU 0`` will be added.
 
-The defines names are automatically prepended with ``HAVE_``, capitalized and
+The defined names are automatically prepended with ``HAVE_``, capitalized and
```
Comment (Akemi) [inline DOCS/waf-buildsystem.rst:71]: the define here refers to the #define directive. so shouldn't it be just "The define names" or "The #define names", but definitely not defined i think.

File: DOCS/contribute.md (hunk 1 above)
Comment (Akemi) [reply DOCS/contribute.md:18]: apparently that is a german thing and there is no plural of information in english.

File: DOCS/contribute.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ Write good commit messages
   information to test the original bug. The old bug might be reintroduced while
   fixing the new bug.
 
-  The commit message must be wrapped on 72 characters per line, because git
+  The commit message must be wrapped in 72 characters per line, because git
```
Comment (haasn) [inline DOCS/contribute.md:84]: I think the correct convention is "wrapped *to* 72 characters".

File: DOCS/waf-buildsystem.rst [hunk 4]
```diff
@@ -17,20 +17,20 @@ shortcomings:
...
+   any single case. That brings us to...
 
 2) --enable-feature has to be overridden by the user and helps them understand that
    they have libraries missing and should install them for the feature to be
    enabled.
 
-3) Must be customizable, hackable, pleasant to the developer eyes and to work
-   with in general.
+3) Must be customizable, hackable, pleasant to the developer's eyes and to work
+   within general.
```
Comment (haasn) [inline DOCS/waf-buildsystem.rst:28]: This is wrong on two accounts:

1. Should be `developers'`, for consistency with your other plural possessive.
2. `within` is actually `with in`, or possibly `with, in` for clarity.

File: DOCS/contribute.md (hunk 3 above)
Comment (LaserEyess) [inline DOCS/contribute.md:84]: "wrapped to" is much more natural English.

File: DOCS/contribute.md [hunk 5]
```diff
@@ -120,10 +120,10 @@ Touching user-visible parts may require updating the mpv docs
 -------------------------------------------------------------
 
 - Most user-visible things are normally documented in DOCS/man/. If your commit
-  touches documented behavior, list of sub-options, etc., you need to adjust the
+  touches documented behavior, a list of sub-options, etc., you need to adjust the
   documentation.
 - These changes usually go into the same commit that changes the code.
-- Changes to command line options (addition/modification/removal) must be
+- Changes to command-line options (addition/modification/removal) must be
```
Comment (LaserEyess) [inline DOCS/contribute.md:126]: "command line" as two words is a lot more common and more well understood. However, the hyphen isn't *technically* wrong, I guess. Even wikipedia has the hyphen in their article.

File: DOCS/waf-buildsystem.rst [hunk 6]
```diff
@@ -44,11 +44,11 @@ tailored to the project's specific needs.
...
 =========================================
 
-To some extents mpv has a custom build system written on top of waf. This
-document will not go over the standard waf behaviour as that is documented in
+To some extent mpv has a custom build system written on top of waf. This
+document will not go over the standard waf behavior as that is documented in
 the `Waf book <http://docs.waf.googlecode.com/git/book_17/single.html>`_.
 
-All of the configuration process is handled with a declarative approach. Lists
+All of the configuration processes are handled with a declarative approach. Lists
```
Comment (haasn) [inline DOCS/waf-buildsystem.rst:51]: Not sure how this plural is being justified. "The entirety of the configuration process is" may have been closer to the original intent here. Not that it really matters.

**Review by haasn — CHANGES_REQUESTED** [review]
Mostly harmless.

**Comment by Akemi** [comment]
from what i gathered just now we would prefer to have all commits squashed into one. possible commit message suffixes could be "docs:" or "man:". former is probably more appropriate.

**Comment by LaserEyess** [comment]
I think most of the hyphen changes are somewhat arbitrary. The only one that has an actual dictionary entry is "long-term", according to Webster's at least. I didn't comment on the other ones because technically they do fit the [APA style guide for hyphenation.](https://apastyle.apa.org/style-grammar-guidelines/spelling-hyphenation/hyphenation) However, I think doing something like that you might as well go through all of `DOCS/man/*` because I'm sure there's inconsistencies there as well.

File: DOCS/contribute.md (hunk 1 above)
Comment (donno2048) [reply DOCS/contribute.md:18]: There is no plural for uncountable noun

File: DOCS/waf-buildsystem.rst (hunk 2 above)
Comment (donno2048) [reply DOCS/waf-buildsystem.rst:71]: Ok, but wouldn't it make more sense to use ``define`` in this context?

File: DOCS/waf-buildsystem.rst (hunk 4 above)
Comment (donno2048) [reply DOCS/waf-buildsystem.rst:28]: Oops, it's really not `within` what a mistake...

File: DOCS/waf-buildsystem.rst (hunk 6 above)
Comment (donno2048) [reply DOCS/waf-buildsystem.rst:51]: "All the" makes no much sense in an individual noun

Comment (donno2048) [reply DOCS/waf-buildsystem.rst:51]: "All the" makes no much sense in an individual noun

File: DOCS/waf-buildsystem.rst (hunk 2 above)
Comment (Akemi) [reply DOCS/waf-buildsystem.rst:71]: if you meant what i first suggested just with formatting, sure.

**Comment by donno2048** [comment]
> The only one that has an actual dictionary entry is "long-term", according to Webster's at least.

Read [my comment](https://github.com/mpv-player/mpv/pull/8748#discussion_r616156066)

**Comment by donno2048** [comment]
Moved to #8757

**Pull request closed without merging**